src/mlpack/tests/loss_functions_test.cpp

```diff
@@ -897,3 +898,53 @@ TEST_CASE("MeanAbsolutePercentageErrorTest", "[LossFunctionsTest]")
   REQUIRE(output.n_cols == input.n_cols);
   CheckMatrices(output, expectedOutput, 0.1);
 }
+
+/*
+ * Simple test for the Triplet Margin Loss function.
+ */
+BOOST_AUTO_TEST_CASE(TripletMarginLossTest)
```

Comment:
```suggestion
TEST_CASE("TripletMarginLossTest")
```
We have removed boost dependency from tests. Also name must be within double quotes.